crawler/lookingglass_crawler/spiders/media_spider.py

```
@@ -107,16 +107,18 @@ def start_requests(self):
                 )
+            except (ValueError, TypeError) as e:
+                self.logger.exception('Failed to create request for %s', url)
             except Exception as e:
-                # Log errors at error level only
-                self.logger.error(f'Failed to create request for {url}: {e}')
+                self.logger.exception('Unexpected error for %s: %s', url, e)
 
     def handle_error(self, failure):
         """Handle request errors and continue crawling."""
-        # Don't log every failed request - too noisy
         self.logger.error(f"Request failed: {failure.request.url} - {failure.value}")
```

Ruff (0.14.7)

Logging statement uses f-string

(G004)